diff --git a/node_modules/preact/src/index.d.ts b/node_modules/preact/src/index.d.ts
index 4c0ab0e..a4a3937 100644
--- a/node_modules/preact/src/index.d.ts
+++ b/node_modules/preact/src/index.d.ts
@@ -94,6 +94,7 @@ export interface FunctionComponent<P = {}> {
 export interface FunctionalComponent<P = {}> extends FunctionComponent<P> {}
 
 export interface ComponentClass<P = {}, S = {}> {
+	__preactClass: undefined;
 	new (props: P, context?: any): Component<P, S>;
 	displayName?: string;
 	defaultProps?: Partial<P>;
@@ -138,6 +139,8 @@ export interface Component<P = {}, S = {}> {
 }
 
 export abstract class Component<P, S> {
+	static __preactClass: undefined;
+
 	constructor(props?: P, context?: any);
 
 	static displayName?: string;
